### DISCUSSION TITLE
question for filters

### DISCUSSION THREAD

**Discussion by Couleyr** [mboquien/cigale#43] [Q&A]
In X-CIGALE, what are the filters corresponding to VISTA (Visible and Infrared Survey Telescope for Astronomy ) bands (Z, Y, J, H, K)?

**Comment by mboquien** [comment]
Hello. This would be `vista.vircam.Z`, `vista.vircam.Y`, `vista.vircam.J`, `vista.vircam.H`, and `vista.vircam.Ks`.

**Reply by Couleyr** [reply]
I noticed that there are only vista.vircam.z band in the filters folder (database_builder/filters), and there are no other four. 
The remaining four are in file database_builder/filters_gazpar/vista/vircam.
Are the names of the remaining four also the same when they are called (such as vista. vircam. Z)?

**Comment by mboquien** [comment]
You can get the list of filters in the database with `pcigale-filters list`. The first column gives the name as it should be used in cigale. For VISTA, they are the names I mentioned earlier.

**Reply by Couleyr** [reply]
Ok, thank you very much for your help.
